storybook/src/docs/form/components/NumberField.stories.mdx

```diff
@@ -0,0 +1,27 @@
+import { Meta, Canvas, Story, Source } from "@storybook/addon-docs";
+
+<Meta title="Docs/Form/Components/NumberField" />
+
+# Number Field
+
+The NumberField returns formatted numbers according to local standard formatting.
+
+## Basic usage
+
+<Canvas>
+    <Story id="stories-form-components--numberfield" />
+</Canvas>
+
+## Usage with decimals
+
+The decimals prop is used to define, how many decimal digits should be displayed.
```

Comment:
It's not just used to define how many are displayed, but primarily how many the value has.